Sources/Core/Utilities/ExifManager/ExifImageSource.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  ExifImageSource.swift
+//  PovioKit
+//
+//  Created by Marko Mijatovic on 17/02/2023.
+//  Copyright © 2023 Povio Inc. All rights reserved.
+//
+
+import Foundation
+
+public enum ExifImageSource {
+  case url(URL)
+  case data(Data)
```

Comment:
Should we also accept `UIImage`?

Reply:
I can add UIImage, but to get the EXIF data, I need to convert it to the Data with `pngData()` (or `jpegData(compressionQuality)`) which will cause a loss of data in the process.